src/pages/instantMessenger/messages/messages.js

```diff
@@ -0,0 +1,33 @@
+import Handlebars from 'handlebars';
+import template from './messages.tmpl';
+import './messages.scss';
+import getHeader from './header/header';
+import getFooter from './footer/footer';
+
+export default () => {
+    let compiled = Handlebars.compile(template);
+   
+    let data = {
+        header: getHeader(),
+        // header: '',
+        footer: getFooter(),
+        messages: [
+            {
+                author: "Андрей",
+                text: "О-ля-ля, какие люди"
+            },
```

Comment:
До этого у тебя были запятые в конце последних строк. Так действительно лучше делать, потому что это положительно сказывается на "диффах" в системах контроля версий
```suggestion
            {
                author: "Андрей",
                text: "О-ля-ля, какие люди",
            },
```